tests/kw_env_test.sh

```diff
@@ -20,6 +20,9 @@ setUp()
   # Create fake .kw folder
   mk_fake_kw_folder "$TEST_PATH"
 
+  # Copy strings
+  cp -r "${ORIGINAL_PATH}/etc/strings" "$TEST_PATH"
```

Comment:
Nitpick: maybe copy only the `etc/strings/env.txt` file instead of the whole directory.

Reply:
Good idea.